go/udpa/data/orca/v1/orca_load_report.pb.go

```diff
@@ -0,0 +1,129 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I think GH is ignoring these, but maybe we should have something like https://github.com/envoyproxy/envoy/blob/master/.gitattributes#L2 to ensure consistent CLI and GH view of the generated files.

Reply:
Done.